californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java

```diff
@@ -679,17 +679,20 @@ private void reject(final RawData raw, final MessageFormatException cause) {
 			rst.setToken(new byte[0]);
 			rst.setDestination(raw.getAddress());
 			rst.setDestinationPort(raw.getPort());
-			for (MessageInterceptor interceptor : interceptors) {
```

Comment:
OK.

But considering my "deep analysis" below, callng reject from reject will cause a message sent to port 0.

And there was a "idea",that in one case the RST could b canceled, in the other not.
(see "rst.isCanceled()"). To be frank, I don't know the background of that difference and may be we don't need it. But then I would prefer to have a explizit statementof changing this.
Otherwise, someone would habe it hard to see, if this was a intended or accidently change.